pkg/driver/controller.go

```diff
@@ -65,6 +65,8 @@ type controller struct {
 	zfsNodeInformer cache.SharedIndexInformer
 
 	volumeLock *volumeLock
+
+	csi.UnimplementedControllerServer
```

Comment:
This type should be member of the CSIDriver type?

Reply:
No, it's different for different services shouldn't be a part of the common type.